pdns/recursordist/rec-main.cc

```diff
@@ -1294,6 +1294,14 @@ static ProxyMappingStats_t& operator+=(ProxyMappingStats_t& a, const ProxyMappin
   return a;
 }
 
+static RemoteLoggerStats_t& operator+=(RemoteLoggerStats_t& a, const RemoteLoggerStats_t& b)
+{
+  for (const auto& [key, entry] : b) {
```

Comment:
I did not know this kind of pattern matching worked! 🥳

Reply:
A lot more is possible, see https://en.cppreference.com/w/cpp/language/structured_binding